embedded-graphics/src/fonts/font24x32.rs

```diff
@@ -0,0 +1,243 @@
+use crate::fonts::font_builder::{FontBuilder, FontBuilderConf};
+
+#[derive(Debug, Copy, Clone)]
+pub enum Font24x32Conf {}
+impl FontBuilderConf for Font24x32Conf {
+    const FONT_IMAGE: &'static [u8] = include_bytes!("../../data/font24x32_1bpp.raw");
+    const CHAR_HEIGHT: u32 = 32;
+    const CHAR_WIDTH: u32 = 24;
+    const FONT_IMAGE_WIDTH: u32 = 960;
+    fn char_offset(c: char) -> u32 {
+        let fallback = '?' as u32 - ' ' as u32;
+        if c < ' ' {
+            return fallback;
+        }
+        if c <= '~' {
+            return c as u32 - ' ' as u32;
+        }
+        fallback
+    }
+}
+
+/// 24x32 pixel monospace font
+///
+/// There is also the [`text_24x32`] macro to provide an easier to use interface.
+///
+/// The 24x32 font is just a scaled up version of the 12x16 font, so [![here is the 12x16 font spritemap screenshot](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)
```

Comment:
This looks a bit odd when rendered out as the link text is actually the image caption:

![image](https://user-images.githubusercontent.com/800791/64919872-09c69e80-d7a8-11e9-9703-d3bf118416ef.png)

I'd suggest this instead:

```suggestion
/// The 24x32 font is a 2x scaling of the [12x16 font].
///
/// The 12x16 font sprite is shown here for reference:
///
/// [![12x16 font spritemap screenshot](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)
```

With the link to the `12x16 font` added at the bottom of the doc comment:

```diff
+ /// [12x16 font]: ./type.Font12x16.html
```

The rendered doc then looks like this, which I think is much better:

![image](https://user-images.githubusercontent.com/800791/64919907-86597d00-d7a8-11e9-828c-27b1aeb42625.png)

Feel free to come up with something different if you think this can be improved further.